packages/portfolio-contract/test/portfolio.contract.test.ts

```diff
@@ -1,42 +1,73 @@
 // prepare-test-env has to go 1st; use a blank line to separate it
 import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
-import type { CoinSDKType } from '@agoric/cosmic-proto/cosmos/base/v1beta1/coin.js';
-import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
-import { heapVowE as VE } from '@agoric/vow/vat.js';
+import { MsgLock } from '@agoric/cosmic-proto/noble/dollar/vaults/v1/tx.js';
+import { MsgSwap } from '@agoric/cosmic-proto/noble/swap/v1/tx.js';
+import type { Installation } from '@agoric/zoe';
 import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
 import { E, passStyleOf } from '@endo/far';
-import { Nat } from '@endo/nat';
 import { M, mustMatch } from '@endo/patterns';
+import type { ExecutionContext } from 'ava';
 import { createRequire } from 'module';
-import { ChainAddressShape } from '@agoric/orchestration';
-import { buildVTransferEvent } from '@agoric/orchestration/tools/ibc-mocks.js';
-import { commonSetup } from './supports.js';
+import { makeUSDNIBCTraffic } from './mocks.ts';
+import { makeTrader } from './portfolio-actors.ts';
+import { setupPortfolioTest } from './supports.ts';
+import { makeWallet } from './wallet-offer-tools.ts';
 
 const nodeRequire = createRequire(import.meta.url);
 
-const contractName = 'myOrchContract';
+const contractName = 'ymax0';
```

Comment:
_question_

Just curious, the 0 implies in the future we could have different iterations of this contract?

Reply:
Yes... I don't want to use up the name ymax with this half-baked stuff.